### PR TITLE
proof-of-concept version of role-based access control server

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Closes: https://github.com/oceanprotocol/RBAC-Server/issues/1

Changes proposed in this PR:
- Building a proof-of-concept (POC) version of the Ocean role-based access control (RBAC) server.

### CODE REVIEW

**Comment by jamiehewitt15** [comment]
@alexcos20 Could you have a look at this and let me know if it is on the right lines in terms of a starting point? At the moment I've just put some example users in a json but we can build on this so that it requests the user roles from other places (smart contracts, 3box, Ldap etc.)

**Review by alexcos20 — COMMENTED** [review]
awesome.
Can we switch to POST and events sent as application/json?

**Comment by jamiehewitt15** [comment]
I've changed the request to POST and the response is now sent as json

File: README.md [hunk 1]
```diff
@@ -1 +1,145 @@
...
+
+```Bash
+npm run start
+```
+
+## Sending requests
+
+- Download and install [Postman](https://www.postman.com/downloads/)
+
+- Send a GET request to `http://localhost:3000`
```
Comment (0x3bfc) [inline README.md:32]: POST

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "rbac-server",
+  "version": "0.0.0",
```
Comment (0x3bfc) [inline package.json:3]: version ?

File: README.md (hunk 1 above)
Comment (jamiehewitt15) [reply README.md]: Thanks for the feedback, I've updated this

File: package.json (hunk 2 above)
Comment (jamiehewitt15) [reply package.json]: Thanks, I've added the version number

**Review by 0x3bfc — APPROVED** [review]
LGTM
Thanks @jamiehewitt15

**Pull request merged**